server/game/core/Constants.ts

```diff
@@ -194,6 +194,7 @@ export enum EventName {
     OnRegroupPhaseReadyCards = 'onRegroupPhaseReadyCards',
     OnRoundEnded = 'onRoundEnded',
     OnRoundEndedCleanup = 'onRoundEndedCleanup',
+    OnScry = 'onScry',
```

Comment:
Missed this in the rename

Reply:
changed it to onLookMoveDeckCardsTopOrBottomSystem